src/components/Restoration/YourCompanySummary.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-card flat id="restoration-summary">
+    <!-- Section Header -->
+    <div class="summary-header px-4 mb-2 rounded-t">
+      <v-row no-gutters>
+        <v-col cols="9">
+          <img class="header-icon" src="@/assets/images/currency-usd-circle.svg">
+          <label class="summary-title">Your Company</label>
+        </v-col>
+      </v-row>
+    </div>
+    <!-- Business Name -->
+    <template>
+      <v-divider class="mx-4" />
+      <div class="section-container business-name-summary">
+        <v-row no-gutters>
+          <v-col cols="3">
+            <label><strong>Restoration Type</strong></label>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedExtendRestorationFiling">
+            <div class="font-weight-bold">Limited Restoration Extension</div>
+            <div>Extension Time: {{ getFormattedExpiryText }}</div>
+          </v-col>
+          <v-col cols="8" class="mt-n1" v-if="isLimitedConversionRestorationFiling">
+            <div class="font-weight-bold">Conversion to Full Restoration</div>
+            <div>[TODO - Applicant's relationship: Director, Shareholder]</div>
```

Comment:
This block isn't part of the current ticket, but I'm adding a placeholder to be replaced at a later date.